api/tests/Feature.FormTemplates.UnitTests/Endpoints/AssignEndpointTests.cs

```diff
@@ -0,0 +1,129 @@
+        _electionRoundFormTemplateRepository = Substitute.For<IRepository<ElectionRoundFormTemplate>>();
+        _endpoint = Factory.Create<Endpoint>(
+            _electionRoundRepository,
+                                    _formTemplateRepository,
+                                _electionRoundFormTemplateRepository);
+    }
+    
+    [Fact]
+    public async Task DeleteRangeAsync_Should_Return_NoContent()
+    {
+        // Arrange
+        _electionRoundRepository.GetByIdAsync(Arg.Any<Guid>()).Returns(new ElectionRoundAggregateFaker());
+        _formTemplateRepository.CountAsync(Arg.Any<ListFormTemplatesByIds>()).Returns(1);
+        
+        var electionRoundId = Guid.NewGuid();
+        var templateIdToKeep = Guid.NewGuid();
+        var templateIdToDelete = Guid.NewGuid();
+        
+        _electionRoundFormTemplateRepository
+            .ListAsync(Arg.Any<ListElectionRoundFormTemplateSpecification>())
+            .Returns(new List<ElectionRoundFormTemplate>
+            {
+                new ElectionRoundFormTemplateAggregateFaker( electionRoundId, templateIdToKeep ),
+                new ElectionRoundFormTemplateAggregateFaker( electionRoundId, templateIdToDelete ),
+            });
+        
+        
+        // Act
+        var request = new Request
+        {
+            ElectionRoundId = electionRoundId,
+            FormTemplateIds = [templateIdToKeep]
+        };
+        
+        var response = await _endpoint.ExecuteAsync(request,default);
+        
+        // Assert
+        response.Should().BeOfType<Results<NoContent, NotFound>>()!
+            .Which!
+            .Result.Should().BeOfType<NoContent>();
+
+        await _electionRoundFormTemplateRepository.Received(1)
+            .DeleteRangeAsync(Arg.Is<IEnumerable<ElectionRoundFormTemplate>>(
+                x => x.Count() == 1 
+                     && x.Any(y => y.FormTemplateId == templateIdToDelete)), Arg.Any<CancellationToken>());
+    }
+    
+    
+    [Fact]
+    public async Task AddRangeAsync_Should_Return_NoContent()
+    {
+        // Arrange
+        _electionRoundRepository.GetByIdAsync(Arg.Any<Guid>()).Returns(new ElectionRoundAggregateFaker());
+        _formTemplateRepository.CountAsync(Arg.Any<ListFormTemplatesByIds>()).Returns(2);
+        _electionRoundFormTemplateRepository
+            .ListAsync(Arg.Any<ListElectionRoundFormTemplateSpecification>()).Returns([]);
+        
+        
+        // Act
+        var request = new Request { FormTemplateIds = [Guid.NewGuid(), Guid.NewGuid()]};
+        
+        var response = await _endpoint.ExecuteAsync(request,default);
+        
+        // Assert
+        response.Should().BeOfType<Results<NoContent, NotFound>>()!
+            .Which!
+            .Result.Should().BeOfType<NoContent>();
+
+        await _electionRoundFormTemplateRepository.Received(1)
+            .AddRangeAsync(Arg.Is<IEnumerable<ElectionRoundFormTemplate>>(x => x.Count() == 2));
```

Comment:
make sure formTemplateIds from request are inserted same as in the test with `DeleteRangeAsync`